tutorials/fundamentals/trim.md

```diff
@@ -0,0 +1,26 @@
+To trim characters from a string in JavaScript, you must utilize the `replace()` function.
+`trim()` is only useful for removing whitespace characters which is why `replace()` needs to be used instead.
+`replace()` takes two arguments:
+- a regular expression denoting what to take out
+- a string denoting what to put in.
+```javascript
+let example = 'aaaaaaaaaaaaaaaaaaaaBBBaaaaaaaaaaaaaaaaaaaa';
+example.replace(/^a+/, 'f').replace(/a+$/, 'g'); // fBBBg
```

Comment:
make this `replace(/^a+/, '')` because this tutorial is about trimming (removing) leading/trailing chars, not replacing them with something else.